shoparize.php

```diff
@@ -70,6 +70,21 @@ function shoparize_after_purchase_action($order_id)
     }
 
     add_action('woocommerce_thankyou', 'shoparize_after_purchase_action', 10, 1 );
+
+    function shoparize_after_purchase_action_by_order_key_parameter($content) {
+        $order_key = $_GET['order_key'] ?? $_GET['key'] ?? false;
+        if ($order_key) {
```

Comment:
Why not to merge two ifs in one?

Reply:
I did it as this because one flow uses 'key' for the order id parameter in URL at the thank you page, and another flow uses 'order_key' to define a created order